libr/debug/Makefile

```diff
@@ -43,6 +43,12 @@ OBJS+= p/${COREDUMP_OBJS}
 endif
 endif
 
+ifneq (,$(findstring bsd,$(OSTYPE))$(findstring dragonfly,$(OSTYPE)))
+ifneq (${NATIVE_OBJS},)
+OBJS+= p/${NATIVE_OBJS}
```

Comment:
uhm i dont think this will work if native objs is more than one word. use $(addprefix instead)